lang/cs/Org.Apache.REEF.IO/FileSystem/AzureBlob/AzureBlockBlobFileSystem.cs

```diff
@@ -191,7 +191,12 @@ public IEnumerable<Uri> GetChildren(Uri directoryUri)
         /// <returns>The URI to the remote file</returns>
         public Uri CreateUriForPath(string path)
```

Comment:
This needs to be re-evaluated. The CreateUriForPath is not doing any sanitization of the provided path. Blobs have very strict naming conventions for containers. All of which should be accounted for here.

Reply:
Added checks using Azure Storage apis